authserver/api/static/css/style.css

```diff
@@ -1,51 +1,80 @@
-html,
-body {
-  height: 100%;
```

Comment:
seems like 
```
html {
  height: 100%;
}
body {
  min-height: 100%;
}
```
is best practice for this. https://stackoverflow.com/questions/6654958/make-body-have-100-of-the-browser-height
then you can set `.login` height to 100%. But yeah besides that and setting margin: 0; and padding:0 on `body` to get rid of some default browser stylings that can interfere with design, you don't style `body` with flex box or anything to keep it as unopinionated as possible

Reply:
done: https://github.com/brighthive/authserver/blob/e0dd4b2c3ca15c36f48e1af8562fdf0b772e14b3/authserver/api/static/css/style.css#L7-L15